core/trino-main/src/main/java/io/trino/execution/TaskManagerConfig.java

```diff
@@ -286,9 +286,9 @@ public int getMaxWorkerThreads()
 
     @LegacyConfig("task.shard.max-threads")
     @Config("task.max-worker-threads")
-    public TaskManagerConfig setMaxWorkerThreads(int maxWorkerThreads)
+    public TaskManagerConfig setMaxWorkerThreads(String maxWorkerThreads)
```

Comment:
Can we add `@ConfigDescription` and documentation ?
Maybe also deprecate `task.shard.max-threads` in another commit

Reply:
> Maybe also deprecate `task.shard.max-threads` in another commit

that's orthogonal, i believe?